controllers/transforms_test.go

```diff
@@ -844,6 +849,111 @@ func TestTransformDCGMExporter(t *testing.T) {
 	}
 }
 
+func TestTransformDCGM(t *testing.T) {
+	limits := corev1.ResourceList{
+		corev1.ResourceCPU:    resource.MustParse("100m"),
```

Comment:
I see these variables declared multiple times. We could move them out of the local scope and make it global, so that all we declare them once and have all test cases use the one single global var